README.md

```diff
@@ -41,6 +42,22 @@ Use in your template.
 ```
 
 Where `@page` is a `%Scrivener.Page{}` struct returned from `Repo.paginate/2`.
+This means you need to add some code in your controller like:
+
+```elixir
+page = MyApp.User
+  |> where([p], p.age > 30)
+  |> order_by([p], desc: p.age)
+  |> MyApp.Repo.paginate(params)
+
+render conn, :index,
+  page: page, # add a line like this.
+  users: page.entries,
+  page_number: page.page_number,
+  page_size: page.page_size,
+  total_pages: page.total_pages,
+  total_entries: page.total_entries
```

Comment:
Lets leave most of this info out so users don't have to divide between what this package does and what other parts of phoenix/ecto do. Just use `page` and call it `users` since the page is enumerable now (meaning you don't need separate `users` variable, you can just do `for user <- page do`).